src/machine/machine_dummy.go

```diff
@@ -25,3 +25,7 @@ func (p GPIO) Configure(config GPIOConfig) {
 
 func (p GPIO) Set(value bool) {
 }
+
+func (p GPIO) Get() (value bool) {
+	return
```

Comment:
It's not entirely obvious when looking at the code that this function returns false (a dummy value). It would be more obvious to do this:
```go
func (p GPIO) Get() bool {
    return false
}
```

Reply:
Done.